src/test/java/org/broadinstitute/hellbender/tools/spark/sv/integration/DiscoverVariantsFromContigAlignmentsSAMSparkIntegrationTest.java

```diff
@@ -35,14 +35,13 @@
         String getCommandLineNoApiKey() {
             return  " -R " + SVIntegrationTestDataProvider.reference_2bit +
                     " -I " + SVIntegrationTestDataProvider.TEST_CONTIG_SAM +
-                    " -O " + outputDir + "/variants.vcf" +
-                    " --fastaReference " + SVIntegrationTestDataProvider.reference;
+                    " -O " + outputDir + "/variants.vcf";
         }
 
     }
 
     @DataProvider(name = "discoverVariantsFromContigAlignmentsSparkIntegrationTest")
-    public Object[][] createTestData() throws IOException {
```

Comment:
Maybe due to my lack of java knowledge, but why was the "throws IOException" removed?

Reply:
For record keeping: my version of IDE indicates the function (or actually the function it calls into) now doesn't throw `IOException` any more, so I removed it.